### DISCUSSION TITLE
Wiki: configuration file is not easy to find

### DISCUSSION THREAD

**Discussion by moviuro** [dani-garcia/vaultwarden#2616] [Ideas]
I recently installed `vaultwarden` on my FreeBSD server using its pkg utility (binary package management).

* I didn't understand that vaultwarden would run as a daemon from browsing the wiki. Did I miss something?
* I couldn't find any manpage or help pointing to the `/usr/local/etc/rc.conf.d/vaultwarden.sample` file which holds vaultwarden settings (and that I should copy as `/usr/local/etc/rc.conf.d/vaultwarden`). I am unsure about where this information should be added to the wiki (Configuration? Third-party package?). The file in `rc.conf.d` is read when starting up the vaultwarden service.

**Comment by BlackDex** [comment]
The most active maintainers of Vaultwarden do not maintain those binary packages. We only build docker images and distribute those to Docker Hub.

Because of this, we have no clue on how the initial configuration needs to be done for this specific package for FreeBSD.

The only thing i can point you to are https://github.com/dani-garcia/vaultwarden/wiki and https://github.com/dani-garcia/vaultwarden/blob/b64cf27038f04368af8f25aa80782d37471e6303/.env.template .

You are free to update the wiki with comments on how to configure it for that package, which is described here: https://github.com/dani-garcia/vaultwarden/wiki/Third-party-packages

**Reply by moviuro** [reply]
Well, I added details to the [FreeBSD section](https://github.com/dani-garcia/vaultwarden/wiki/Third-party-packages#freebsd), I hope it helps the next ones.
